components/ee/ws-scheduler/pkg/scheduler/config.go

```diff
@@ -22,6 +22,8 @@ type Configuration struct {
 	StrategyName StrategyName `json:"strategyName"`
 	// DensityAndExperienceConfig is the (optional) config for the DensityAndExperience strategy
 	DensityAndExperienceConfig *DensityAndExperienceConfig `json:"densityAndExperienceConfig,omitempty"`
+	// RAMSafetyBuffer reduces the amount of available RAM per node and is meant to make sure we do not overbook nodes
+	RAMSafetyBuffer string `json:"ramSafetyBuffer,omitempty"`
```

Comment:
can't this be `res.Quantity` like in [scaler](https://github.com/gitpod-io/gitpod/blob/0807589ea373c2ebbe74f41a83ba9ea4a54fb259/components/ee/ws-scheduler/pkg/scaler/config.go#L21)?

Quantity sports `UnmarshalJSON`

Reply:
it can't be because then the field becomes mandatory. There's no good way to have optional quantities in JSON